bin/redis-commander.js

```diff
@@ -414,6 +481,67 @@ function createConnectionObjectFromArgs(argList) {
     if (argList['redis-tls']) {
       connObj.tls = {};
     }
+    if (argList['redis-tls'] &&
+      (argList['redis-tls-ca-cert-file'] || argList['redis-tls-ca-cert']
+        || argList['redis-tls-cert-file'] || argList['redis-tls-cert']
+        || argList['redis-tls-key-file'] || argList['redis-tls-key']
+        || argList['redis-tls-server-name'])) {
+      connObj.tls = {};
+      if (argList['redis-tls-ca-cert-file']) {
+        connObj.tls.ca = fs.readFileSync(argList['redis-tls-ca-cert-file']);
+      }
+      else if (argList['redis-tls-ca-cert']) {
+        connObj.tls.ca = argList['redis-tls-ca-cert'];
+      }
+      if (argList['redis-tls-cert-file']) {
+        connObj.tls.cert = fs.readFileSync(argList['redis-tls-cert-file']);
+      }
+      else if (argList['redis-tls-cert']) {
+        connObj.tls.cert = argList['redis-tls-cert'];
+      }
+      if (argList['redis-tls-key-file']) {
+        connObj.tls.key = fs.readFileSync(argList['redis-tls-key-file']);
+      }
+      else if (argList['redis-tls-key']) {
+        connObj.tls.key = argList['redis-tls-key'];
+      }
+      if (argList['redis-tls-server-name']) {
+        connObj.tls.servername = argList['redis-tls-server-name'];
+      }
+    }
+    if (argList['enable-sentinel-tls']) {
+      connObj.enableTLSForSentinelMode = true;
+    }
+    if (argList['sentinel-tls']) {
+      connObj.sentinelTLS = {};
+    }
```

Comment:
Here i do not really understand the meaning/difference between these two flags "enable-sentinel-tls" and "sentinel-tls"
Does it make sense to only set one of the two? 

Should not be setting "sentinel-tls" (similar to "redis-tls") be enough and we set internally the other option automatically too than?